mmv1/third_party/terraform/services/container/go/node_config.go.tmpl

```diff
@@ -1134,6 +1159,13 @@ func expandWorkloadMetadataConfig(v interface{}) *container.WorkloadMetadataConf
 	return wmc
 }
 
+func expandInsecureKubeletReadonlyPortEnabled(v interface{}) bool {
+	if v == "TRUE" {
```

Comment:
Not technically "expanding" here, I guess, since it's coming in as a string? But I think probably best to keep it named this way for consistency?

Reply:
Yes, expand functions normally turn the config value into the API value.